Cargo.toml

```diff
@@ -34,6 +34,7 @@ members = [
     "primitives/session-manager",
     "primitives/sidechain",
 	"partner-chains-cli",
+    "pallets/native-token-management",
```

Comment:
How do we "manage" this token? Is it the business/product name of the feature?
I know that it is quite late, for this comment. Maybe change could be a separate PR now.

Reply:
The requirements doc calls the feature "native token management". I don't think it's a good name either. Since there's already crates with the management name merged, let's eventually revisit it in a separate PR as you said.